modules/special-rules/pages/super-heavy-weapons.adoc

```diff
@@ -7,48 +8,58 @@ These rules define the effects of super-heavy weapons; see the specific unit dat
 
 == Super-heavy weapons and placement of Blast markers
 
 When you shoot with a detachment that includes super-heavy weapons, you may get to place two lots of Blast markers:
 
 * For any regular Firepower that your detachment brings to bear on the target detachment, place Blast markers as normal.
 * For the super-heavy weapons that your detachment brings to bear on the target detachment, place Blast markers as follows.
-+
-|===
-^|Number of super-heavy weapons brought to bear ^|Number of Blast markers to place
-
-^|1 to 3 ^|*
-^|4 to 8 ^|1
-^|9 or more ^|2
-2+|* = 1 Blast marker, but only if you score at least one hit.
-|===
-+
+
+include::main-rules:partial$blast-marker-table.adoc[]
+
 [NOTE]
 ====
 Count the number of super-heavy weapons that you direct at the target detachment.
-Do not modify this number because of Assault orders, Blast markers on the shooting detachment, or anything similar.
 
-+[TODO: Check and consider the wording around this. I seem to remember that there were some inconsistencies or nuances in how Firepower and super-heavy weapons are suppressed by Blast markers. There may be an important difference between how many we "bring to bear" and how many can actually shoot, that also effects Blast marker placement.]+
+* Do *not* modify this number because of Assault orders, Blast markers on the shooting detachment, or anything similar.
+* Do *not* count super-heavy weapons that add to the detachment's Firepower total, such as {barrage} -- the Firepower component already accounts for any Blast markers that they may add.
 ====
 
 == Super-heavy weapons and suppression by Blast markers
 
-With regular Firepower, Blast markers on the detachment each cause the loss of one point of Firepower in the Shooting phase.
-However, where a detachment has only super-heavy weapons available then each Blast marker will normally suppress one entire super-heavy weapon.
-That is, unless the weapon profile on a unit datasheet says otherwise, treat each super-heavy weapon as having a Firepower value of 1.
+Normally, for each Blast marker on a detachment you must reduce its Firepower by 1.
+However, if the detachment has more Blast markers than Firepower, or has no Firepower, then for each remaining Blast marker you lose the ability to shoot with one super-heavy weapon.
+You may choose which super-heavy weapons are suppressed.
+// [TODO: Define suppression properly, perhaps in blast-markers.adoc -- as it is we use the term and imply its meaning, but haven't actually made it explicit.]
+
+NOTE: The entire super-heavy weapon is suppressed, even if it has multiple shots like {pulsar}, or adds to a detachment's Firepower like {barrage}.
+
+== Super-heavy weapons in firefights
+
+Where a super-heavy weapon doesn't have a basic Firepower value, but is in a position to contribute to a firefight, it contributes 1 Firepower.
+{blank}[TODO: Is it just firefights? Do SHWs count as FP1 in any other situations, like supporting fire in close combat?]
```

Comment:
I can't think of any. In close combats, each supporting unit adds +1 to the Assault total, without mention of Firepower or supporting 'fire'.

Separate from that, the special rule for war engines supporting close combats isn't weapon-based anyway, but is half the current DC.

Reply:
Gotcha. Something was nagging at my memory, so thought I'd better double check. That's fine then, I'll delete my TODO query here.